ebmlite/cli.py

```diff
@@ -36,7 +36,7 @@ def load_files(args):
     output = os.path.realpath(os.path.expanduser(args.output))
     if os.path.exists(output) and not args.clobber:
         errPrint("Output file exists: %s" % args.output)
-    with open(output, 'wb') as out:
+    with open(output, ('wb' if binary_output else 'w')) as out:
```

Comment:
Why not make 'mode' a parameter, and just supply either `"w"` or `"wb"` as the argument?

Reply:
because it removes the possibility of misusing the function with `mode='r'`, or god forbid `mode=5`.

why should I make a mode parameter? brevity?